src/v2/Components/Pagination/index.tsx

```diff
@@ -29,6 +30,7 @@ export const Pagination: React.FC<Props> = ({
   getHref: __getHref__,
   onClick = _cursor => ({}),
   onNext = () => ({}),
+  offset,
```

Comment:
Should set the default here

```suggestion
  offset = 40,
```